packages/fast-components-styles-msft/src/image/index.ts

```diff
@@ -2,6 +2,9 @@ import { IDesignSystem } from "../design-system";
 import { ComponentStyles } from "@microsoft/fast-jss-manager";
 import { IImageClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 
+
+
+

```

Comment:
@nicholasrice can we remove these three empty lines?

Reply:
never!